### PR TITLE
add new ioctl alias remove command and test

### PR DESCRIPTION
**Pull request by zeromake**
#1771 
@CoderZhi

### CODE REVIEW

File: ioctl/newcmd/alias/aliasremove.go [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+// NewAliasRemove represents the removes alias command
+func NewAliasRemove(c ioctl.Client) *cobra.Command {
+	use, _ := c.SelectTranslation(removeUses)
+	short, _ := c.SelectTranslation(removeShorts)
+	invalidAlias, _ := c.SelectTranslation(removeInvalidAlias)
+	marshalError, _ := c.SelectTranslation(removeMarshalError)
+	writeError, _ := c.SelectTranslation(removeWriteError)
+	result, _ := c.SelectTranslation(removeResult)
+	conf := c.Config()
```
Comment (CoderZhi) [inline ioctl/newcmd/alias/aliasremove.go]: Move into function

File: ioctl/newcmd/alias/aliasremove_test.go [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+	cfg := config.Config{
+		Aliases: map[string]string{
+			"a": "io1uwnr55vqmhf3xeg5phgurlyl702af6eju542sx",
+			"b": "io1uwnr55vqmhf3xeg5phgurlyl702af6eju542sx",
+			"c": "io1uwnr55vqmhf3xeg5phgurlyl702af6eju542s1",
+		},
+	}
+	client.EXPECT().Config().Return(cfg).AnyTimes()
+	cmd := NewAliasRemove(client)
+	res, err := util.ExecuteCmd(cmd, "a")
```
Comment (CoderZhi) [inline ioctl/newcmd/alias/aliasremove_test.go]: incomplete unit test: how do you verify that the corresponding alias has been removed?

Comment (zeromake) [reply ioctl/newcmd/alias/aliasremove_test.go]: @CoderZhi 
supplementary test

File: ioctl/newcmd/alias/aliasremove_test.go [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+	require.NotNil(t, res)
+	require.NoError(t, err)
+
+	// read config file check aliases
+	conf, err := config.LoadConfig()
+	require.NoError(t, err)
+	_, ok := conf.Aliases["a"]
+	require.Equal(t, ok, false)
```
Comment (CoderZhi) [inline ioctl/newcmd/alias/aliasremove_test.go]: require.False(t, ok)

File: ioctl/newcmd/alias/aliasremove_test.go [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+	_, ok = conf.Aliases["b"]
+	require.Equal(t, ok, true)
```
Comment (CoderZhi) [inline ioctl/newcmd/alias/aliasremove_test.go]: require.True(t, ok)

File: ioctl/newcmd/alias/aliasremove.go [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+			}
+			conf := c.Config()
+			delete(conf.Aliases, alias)
+			out, err := yaml.Marshal(&conf)
+			if err != nil {
+				return output.NewError(output.SerializationError, marshalError, err)
+			}
+			if err := ioutil.WriteFile(config.DefaultConfigFile, out, 0600); err != nil {
+				return output.NewError(output.WriteFileError, fmt.Sprintf(writeError, config.DefaultConfigFile), err)
+			}
```
Comment (CoderZhi) [inline ioctl/newcmd/alias/aliasremove.go]: it would better if we can move these lines into client

Comment (zeromake) [reply ioctl/newcmd/alias/aliasremove.go]: Does it mean adding a new method on the client?
E.g:
``` go
type Client interface {
     SaveConfig(cfg config.Config) error
}
```

Comment (zeromake) [reply ioctl/newcmd/alias/aliasremove.go]: @CoderZhi 
Does it mean adding a new method on the client?
E.g:
``` go
type Client interface {
     SaveConfig(cfg config.Config) error
}
```

**Pull request merged**